data/ecosystems/o/openproduct.toml

```diff
@@ -15,5 +15,8 @@ url = "https://github.com/OpenProduct/openmask-extension"
 [[repo]]
 url = "https://github.com/OpenProduct/openmask-websdk"
 
+[[repo]]
+url = "https://github.com/OpenProduct/jetton-dao-contracts"
```

Comment:
Already part of OpenProduct ecosystem, which is part of TON. Can you remove this one please?